scripts/build/arch/sparc.sh

```diff
@@ -2,4 +2,17 @@
 CT_DoArchTupleValues() {
     # That's the only thing to override
     CT_TARGET_ARCH="sparc${target_bits_64}${CT_ARCH_SUFFIX}"
+
+    # By default, sparc64-*-linux is configured with -mcpu=v9. However,
+    # according to https://sourceware.org/ml/libc-alpha/2005-12/msg00027.html,
+    # "There is no Linux sparc64 port that runs on non-UltraSPARC-I+ ISA CPUs."
+    # There is a patch that would change the default to -mcpu=ultrasparc for
+    # sparc64-*-linux configuration: https://patchwork.ozlabs.org/patch/409424/
+    # but that patch has not been integrated (yet). One concern raised about
+    # this patch was that -mcpu=ultrasparc can suboptimally schedule instructions
+    # for newer SPARC CPUs. So, override to -mcpu=ultrasparc and warn the user.
```

Comment:
Sorry, I'm not entirely sure I grok this explanation. Let me try to
suggest an alternate commit log:

    arch/sparc: better default CPU when targetting Linux

    By default, sparc64-*-linux is configured with -mcpu=v9. However,
    according to https://sourceware.org/ml/libc-alpha/2005-12/msg00027.html:
        "There is no Linux sparc64 port that runs on non-UltraSPARC-I+ ISA
         CPUs."

    v9 is such a "non-UltraSPARC-I+ ISA CPU", so it makes no sense to
    default to v9 when targetting Linux.

    Change the default to ultrasparc, even though it can suboptimally
    schedule instructions for newer SPARC CPUs. See the pending patch:
        https://patchwork.ozlabs.org/patch/409424/

If that's indeed what you meant, feel free to use it. If nopt, please
explain in more details...

Also, the discussion on the proposed patch suggests that ultrasparc3
would be a better default. Why not use it instead?